### PR TITLE
invada-studio-plugins-lv2 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by david0**

### CODE REVIEW

File: Library/Formula/invada-studio-plugins-lv2.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class InvadaStudioPluginsLv2 < Formula
+  desc "LV2 audio plugins suite"
+  homepage "https://launchpad.net/invada-studio/"
+  url "https://launchpad.net/invada-studio/lv2/1.2/+download/invada-studio-plugins-lv2_1.2.0-nopkg.tgz"
+  version "1.2.0"
+  sha256 "c6cac7c32effc6b3052e3b017133244f385ef8e053147859d88eae6facaf7d12"
+  head "lp:invada-studio/lv2", :using=>:bzr
+
+  depends_on "lv2"
```
Comment (mikemcquaid) [inline Library/Formula/invada-studio-plugins-lv2.rb]: I think this formula should probably be part of `lv2` instead as a `resource`

Comment (david0) [inline Library/Formula/invada-studio-plugins-lv2.rb]: That would be possible, but I don't think that this is an modular approach. Many resources might be added to the `lv2` each with different dependencies, different versions (stable/head) making the `lv2` formula very fragile and subject of frequent change. 

Currently there is already a similar formula in homebrew btw: [mda-lv2](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/mda-lv2.rb)

Comment (mikemcquaid) [inline Library/Formula/invada-studio-plugins-lv2.rb]: I don't think plugins make much sense as separate formulae, really. Will let other maintainers say what they think.

File: Library/Formula/invada-studio-plugins-lv2.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class InvadaStudioPluginsLv2 < Formula
+  desc "LV2 audio plugins suite"
+  homepage "https://launchpad.net/invada-studio/"
+  url "https://launchpad.net/invada-studio/lv2/1.2/+download/invada-studio-plugins-lv2_1.2.0-nopkg.tgz"
+  version "1.2.0"
+  sha256 "c6cac7c32effc6b3052e3b017133244f385ef8e053147859d88eae6facaf7d12"
+  head "lp:invada-studio/lv2", :using=>:bzr
```
Comment (bfontaine) [inline Library/Formula/invada-studio-plugins-lv2.rb]: Please use spaces around `=>`.

File: Library/Formula/invada-studio-plugins-lv2.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/invada-studio-plugins-lv2.rb]: I agree with Mike; I think plugins should either be part of the main formula or formulae in a dedicated tap (like e.g. [`homebrew/emacs`](https://github.com/Homebrew/homebrew-emacs)).

File: Library/Formula/invada-studio-plugins-lv2.rb (hunk 2 above)
Comment (bfontaine) [inline Library/Formula/invada-studio-plugins-lv2.rb]: Ping?

**Comment by apjanke** [comment]
Closing this due to inactivity/lack of response. If you'd still like to pursue this, feel free to respond to the previous comments/questions and re-open on the new [`homebrew/core` repo](https://github.com/homebrew/homebrew-core).

**Pull request closed without merging**